pos_order_mgmt/README.rst

```diff
@@ -104,6 +104,7 @@ Contributors
 
 * David Vidal <david.vidal@tecnativa.com>
 * Sylvain LE GAL (https://twitter.com/legalsylvain)
+* Carlos Martínez <carlos@domatix.com>
```

Comment:
You must do on readme/CONTRIBUTORS.rst, not here.

Reply:
Done